ShopperInsights/src/main/java/com/braintreepayments/api/EligiblePaymentsApi.kt

```diff
@@ -8,33 +8,36 @@ internal class EligiblePaymentsApi(
 ) {
     fun execute(request: EligiblePaymentsApiRequest, callback: EligiblePaymentsCallback) {
         val jsonBody = request.toJson()
-        braintreeClient.getConfiguration { configuration, configError ->
+        braintreeClient.getConfiguration { configuration, _ ->
```

Comment:
Out of scope of this PR maybe but I think we are missing an outer layer of error handling - we should be checking that configuration is not null first otherwise calling back `configError` right?

Reply:
i noticed this as well. i'll create a follow up ticket for adding the missing error handling.